commands.go

```diff
@@ -788,6 +788,50 @@ func (c cmdable) Set(ctx context.Context, key string, value interface{}, expirat
 	return cmd
 }
 
+// SetArgs provides arguments for the SetWithArgs command.
+//
+// Mode can be `NX` or `XX` or empty.
```

Comment:
This should be a comment on `Mode string` field. And similar for Zero and Get.

Reply:
I improved the readability